scripts/build

```diff
@@ -1,3 +1,3 @@
 set -e
-echo "jshinting" && node_modules/.bin/jshint Scope.\$watchOnce.js && echo "success"
+echo "jshinting" && npm test && echo "success"
 echo "uglifying" && node_modules/.bin/uglifyjs Scope.\$watchOnce.js --comments -o Scope.\$watchOnce.min.js && echo "success"
```

Comment:
Because you've added the `npm run uglify` script, could we do this here as well? Or even better, get rid of this script file and simply have an `npm run build` script that runs the tests and uglify?